qrexec/policy/admin.py

```diff
@@ -87,7 +88,17 @@ def handle_request(
         Throws PolicyAdminException in case of user error.
         """
 
-        assert all(char in RPCNAME_ALLOWED_CHARSET for char in arg)
+        if service_name in ["policy.Get", "policy.Replace", "policy.Remove"] \
```

Comment:
This missed a few methods that also take file name (all the "include" ones). Anyway, as said in the other comment, it doesn't necessarily needs to be in this place.

Reply:
OK. I will move it to `_get_path`. But the validation within `handle_request` will definitely precede that. Overall this should not be a serious issue since (the fixed) validation within `qubes-policy-editor-gui` will precede all of them.